WORKSPACE

```diff
@@ -1,3 +1,5 @@
+workspace(name = "org_tensorflow_serving")
```

Comment:
Thanks so much for your patience with this. After some more thought, can we make this workspace name "tf_serving"? This simplifies some things for us and keeps the name short.